03_reverseString/reverseString.js

```diff
@@ -1,5 +1,5 @@
 const reverseString = function(string) {
- return string.split('').reverse().join('');
+  return string.split('').reverse().join('');
```

Comment:
Finally, after a bunch of confusion, this is logically returned to the way it was.

I do see a whitespace change crept in, but I think it's for the better (and consistent with other cases), so I'll leave it.